### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by alecandido** [NNPDF/pineko#4] [General]
Hi guys,
GitHub just presented (actually it is presenting) this new tool.

PRs and issues are great, but sometimes we are just discussing about stuffs, I thought that maybe this will be the proper place :smiley:

**Comment by scarrazza** [comment]
Ok, seems like a build-in slack.

**Reply by alecandido** [reply]
Something like that, but much more integrated (they promised), e.g. you should be able to convert discussions in issues when you believe it's relevant.
